selectrum.el

```diff
@@ -97,11 +97,53 @@ See `minibuffer-default-in-prompt-regexps', from which this is derived.")
   :link '(url-link "https://github.com/raxod502/selectrum"))
 
 (defcustom selectrum-num-candidates-displayed 10
-  "Maximum number of candidates which are displayed at the same time.
-The height of the minibuffer will be this number of rows plus one
-for the prompt line, assuming no multiline text."
+  "Maximum number of candidate lines which are displayed.
+Selectrum will display candidates lines up to this number or
+fewer if there are less candidates in total.
+
+For the minibuffer the window height equals this number plus one
+for the prompt line. If `selectrum-display-action' is non-nil
+this option determines the maximal window height but when the
+displaying window height is greater than that all of the
+available height will be used for candidate display."
   :type 'number)
 
+(defun selectrum-display-full-frame (buf _alist)
+  "Display BUF in full frame.
+Can be used as `selectrum-display-action' to display candidates
+in a single window spanning the current frame:
+
+    (setq selectrum-display-action
+        '(selectrum-display-full-frame)."
```

Comment:
Unfortunately, this is needed in order to make the documentation viewer show a normal quote instead of a curly quote:

```suggestion
          \\='(selectrum-display-full-frame)."
```

Reply:
Thanks, I fixed those.